<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDetailsVehicule</class>
 <widget class="QDialog" name="DialogDetailsVehicule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Détails du véhicule</string>
  </property>
  <property name="styleSheet">
   <string>QDialog {
    background-color: #1e1e1e;
    color: #e0e0e0;
}
QLabel {
    color: #e0e0e0;
}
QGroupBox {
    border: 1px solid #3a3a3a;
    border-radius: 5px;
    margin-top: 10px;
    padding-top: 10px;
    color: #e0e0e0;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
}
QPushButton {
    background-color: #252525;
    color: #e0e0e0;
    border: 1px solid #3a3a3a;
    padding: 8px 16px;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #2e2e2e;
    border: 1px solid #0d47a1;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}
QTableWidget {
    background-color: #252525;
    color: #e0e0e0;
    gridline-color: #3a3a3a;
    border: 1px solid #3a3a3a;
}
QTableWidget::item {
    padding: 5px;
}
QHeaderView::section {
    background-color: #2e2e2e;
    color: #e0e0e0;
    padding: 5px;
    border: 1px solid #3a3a3a;
}
QProgressBar {
    border: 1px solid #3a3a3a;
    border-radius: 3px;
    text-align: center;
    background-color: #1e1e1e;
    color: #e0e0e0;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxInfos">
     <property name="title">
      <string>Informations du véhicule</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ID :</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelID">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Type :</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nom :</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelNom">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>État :</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelEtat">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCapacite">
     <property name="title">
      <string>Capacité de charge</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Capacité maximale :</string>
          </property>
          <property name="styleSheet">
           <string>font-weight: bold;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelCapaciteMax">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Capacité utilisée :</string>
          </property>
          <property name="styleSheet">
           <string>font-weight: bold;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelCapaciteUtilisee">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Capacité disponible :</string>
          </property>
          <property name="styleSheet">
           <string>font-weight: bold;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelCapaciteDispo">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarCapacite">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%p%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxColis">
     <property name="title">
      <string>Colis à bord</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Nombre de colis :</string>
          </property>
          <property name="styleSheet">
           <string>font-weight: bold;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelNombreColis">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableColisVehicule">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Masse (kg)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Infos</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnFermer</sender>
   <signal>clicked()</signal>
   <receiver>DialogDetailsVehicule</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
